core/src/main/java/com/klaytn/caver/abi/ABI.java

```diff
@@ -89,6 +102,16 @@ public static String buildFunctionString(ContractMethod method) {
         return result.toString();
     }
 
+    public static String buildFunctionString(String methodName, List<Type> parameters) {
```

Comment:
I think, buildFunctionString function doesn't need actual parameters. Can this function be modified to accept only Type?